buildSrc/src/main/groovy/org/aya/gradle/CommonTasks.groovy

```diff
@@ -54,6 +54,7 @@ final class CommonTasks {
     var task = project.tasks.register('generateNativeImageConfig', GenerateReflectionConfigTask) {
       outputDir = configGenDir
       inputFile = configTemplateFile
+      outputs.upToDateWhen { false }
```

Comment:
Why

Reply:
It keeps showing up-to-date here. So I googled "gradle task always rerun" 😭. Do you have better idea?

Reply:
> It keeps showing up-to-date here. So I googled "gradle task always rerun" 😭. Do you have better idea?

I'll see what I can do

Reply:
I don't understand...

Reply:
I don't think there's anything I can do. Let's keep your changes